testing/BUILD.gn

```diff
@@ -37,6 +41,7 @@ source_set("testing") {
   ]
 
   public_deps = [ ":testing_lib" ]
+  public_configs = [ ":dynamic_symbols" ]
```

Comment:
ldflags don't need to go in public configs to be inherited by the `executable` or `shared_library` target. Source sets are not linked so `ldflags` are implicitly set on the final link step.

Reply:
I was wrong about this. `ldflags`, unlike `libs` on `source_sets` are silently ignored :(